searchcore/src/vespa/searchcore/bmcluster/bm_merge_stats.h

```diff
@@ -0,0 +1,27 @@
+// Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#pragma once
+
+#include <cstdint>
+
+namespace search::bmcluster {
+
+/*
+ * Class containing merge (throttler) stats for a service layer node.
+ */
+class BmMergeStats
+{
+    uint32_t _active;
+    uint32_t _queued;
+    
+public:
+    BmMergeStats();
+    BmMergeStats(uint32_t active, uint32_t queued);
+    ~BmMergeStats();
+    BmMergeStats& operator+=(const BmMergeStats& rhs);
+    bool operator==(const BmMergeStats &rhs) const;
+    uint64_t get_active() const noexcept { return _active; }
```

Comment:
Any reason for the type mismatch between member variable and return value? Should we align them? Also applies for __queued_.

Reply:
No. Fixed.